product_docs/docs/epas/15/reference/sql_reference/02_data_types/range_types.mdx

```diff
@@ -150,31 +150,31 @@ CREATE TYPE floatrange AS RANGE (
 SELECT '[1.234, 5.678]'::floatrange;
 ```
 
-Because `float8` has no meaningful "step", we do not define a canonicalization function in this example.
+Because `float8` has no meaningful step, the example doesn't define a canonicalization function.
 
-If the subtype is considered to have discrete rather than continuous values, the `CREATE TYPE` command should specify a `canonical` function. The canonicalization function takes an input range value, and must return an equivalent range value that may have different bounds and formatting. The canonical output for two ranges that represent the same set of values, for example the integer ranges `[1, 7]` and `[1, 8)`, must be identical. It doesn't matter which representation you choose to be the canonical one, so long as two equivalent values with different formattings are always mapped to the same value with the same formatting. In addition to adjusting the inclusive/exclusive bounds format, a canonicalization function might round off boundary values, in case the desired step size is larger than what the subtype is capable of storing. For instance, a range type over timestamp could be defined to have a step size of an hour, in which case the canonicalization function would need to round off bounds that weren't a multiple of an hour, or perhaps throw an error instead.
+If the subtype is considered to have discrete rather than continuous values, the `CREATE TYPE` command must specify a `canonical` function. The canonicalization function takes an input range value and must return an equivalent range value that might have different bounds and formatting. The canonical output for two ranges that represent the same set of values, for example, the integer ranges `[1, 7]` and `[1, 8)`, must be identical. It doesn't matter which representation you choose to be the canonical one, as long as two equivalent values with different formatting are always mapped to the same value with the same formatting. In addition to adjusting the inclusive/exclusive bounds format, a canonicalization function might round off boundary values, in case the desired step size is larger than what the subtype is capable of storing. For instance, you can define a range type over timestamp to have a step size of an hour. In that case, the canonicalization function needs to round off bounds that weren't a multiple of an hour, or perhaps throw an error instead.
 
-Defining your own range type also allows you to specify a different subtype B-tree operator class or collation to use, so as to change the sort ordering that determines which values fall into a given range.
+Defining your own range type also allows you to specify a different subtype B-tree operator class or collation to use to change the sort ordering that determines which values fall into a given range.
```

Comment:
Maybe change "collation to use to change" to "collation to change" ?

Reply:
```suggestion
Defining your own range type also allows you to specify a different subtype B-tree operator class or collation to change the sort ordering that determines which values fall into a given range.
```